tests/unit/queries/is-test.js

```diff
@@ -0,0 +1,113 @@
+import { test } from 'qunit';
+import { fixture, moduleFor } from '../test-helper';
+import { create, is } from '../../page-object';
+import { test_throws_if_not_multiple } from '../shared';
+
+moduleFor('Unit | Property | .is');
+
+test('returns is value', function(assert) {
+  fixture('<input type="checkbox" checked>');
+
+  let page = create({
+    foo: is(':checked', ':input')
+  });
+
+  assert.equal(page.foo, true);
+});
+
+test("raises an error when the element doesn't exist", function(assert) {
+  let page = create({
+    foo: is(':checked', ':input')
+  });
+
+  assert.throws(() => page.foo, 'Throws element not found error');
+});
```

Comment:
We need to update this test, see https://github.com/san650/ember-cli-page-object/blob/948aec01fe0fd9a94438c63a5380b9d02a8a1ca5/tests/unit/queries/text-test.js#L71-L83